app/src/marketplace/components/GetAccess/GetAccess.tsx

```diff
@@ -36,9 +39,23 @@ export const GetAccess: FunctionComponent<{project: Project}> = ({project}) => {
                 <strong>{formatChainName(firstSalePointChainName)}</strong>
                 {moreSalePoints > 0 && <span> [and on {moreSalePoints} other {moreSalePoints === 1 ? 'chain' : 'chains'}]</span>}
             </p>
-            <Button onClick={() => { purchaseDialog.open({ projectId: project.id }) }}>Get Access</Button>
+            <Button
+                disabled={isBeingPurchased}
```

Comment:
The new flow is traceable. You can tell if a project is being purchased from a mile out.